menu-memory/menu.js

```diff
@@ -0,0 +1,50 @@
+const { dayOfPlay } = require("./../config.json");
+const fs = require('fs');
+const path = require('path');
+const memory = path.resolve(__dirname, "memory.json");
+
+const loadMenuData = async () => {
+	try {
+		return JSON.parse(fs.readFileSync(memory));
+	} catch (e) {
+		return [];
```

Comment:
??

Reply:
Try to read the memory, it it fails then load an empty array. Gotta read on each call to not keep the thing in memory

Reply:
Would it be better to try to create that file if it doesn't exist, and then read it?

Reply:
Why? I will create it later in the save call. So no point creating, reading, and then rewriting. When I could default, then write.

Reply:
The code in the try could fail for other reasons, no? It makes it unclear

Reply:
If there is any error in reading the database there is one solution, loading a default. I guess I don’t see the issue. This is the re-read version of `const data = require(‘file.json’);` I don’t bother splitting up the call because it is so standard (in my head)